src/languageHandler.js

```diff
@@ -338,8 +338,8 @@ export function getCurrentLanguage() {
 
 function getLangsJson() {
     return new Promise((resolve, reject) => {
-        // LANGUAGES_FILE is a webpack compile-time define, see webpack config
-        const url = (typeof LANGUAGES_FILE === "string") ? require(LANGUAGES_FILE) : (i18nFolder + 'languages.json');
+        // Webapp is a webpack resolve alias pointing to the output directory, see webpack config
+        const url = require('Webapp/i18n/languages.json');
```

Comment:
This means the SDK now depends on Riot's Webpack config, which seems unfortunate. Maybe we can wrap this in a `try` and fallback to the `i18nFolder` version for non-Webpack/non-Riot usage?

Reply:
Good point - this also makes the unit tests work without extra config